app/models/concerns/api_action_caching.rb

```diff
@@ -11,8 +11,9 @@ module ApiActionCaching
     include ActionController::Caching::Actions
     include AbstractController::Layouts
 
-    # This config is not passed to the controller automatically with ActionController::Metal
+    # These configs are not assigned to the controller automatically with ActionController::Metal
     self.cache_store = Rails.configuration.cache_store
+    self.perform_caching = Rails.configuration.action_controller.perform_caching
```

Comment:
next step will be to make this an env var or something that we can easily enable/disable :+1: